coalib/processes/Processing.py

```diff
@@ -452,12 +452,16 @@ def yield_ignore_ranges(file_dict):
                                    line_number,
                                    len(file[line_number-1])))
                 elif "ignore " in line:
+                    # When using a plain "ignore" we just want to exclude
+                    # analysis in the following line. And if there's no
+                    # following line, just ignore the ignore-comment itself.
```

Comment:
I personally feel like this comment is unneeded, the min explains it already qute well